scripts/build/Dockerfile.archlinux

```diff
@@ -0,0 +1,45 @@
+FROM archlinux
```

Comment:
Please do not use just a shortname. Use something like `docker.io/library/archlinux:latest
`